.github/workflows/ci.yml

```diff
@@ -13,27 +13,43 @@ on:
 
 env:
   RUST_BACKTRACE: 1
-  PLRUST_TRUSTED_PGRX_OVERRIDE: "pgrx = { path = '/home/runner/work/plrust/plrust/plrust-trusted-pgrx', package='plrust-trusted-pgrx' }"
   # CARGO_LOG: cargo::core::compiler::fingerprint=info # Uncomment this to output compiler fingerprint info
 
 jobs:
-  plrust_tests_arm64:
-    name: aarch64 tests (pg14, amzn2)
-    runs-on: [self-hosted, linux, ARM64, plrust]
+  # A job for running tasks on an Amazon Linux 2 instance, which are self-hosted runners on another cloud service.
+  # The image created for running these tasks already have a majority of the dependencies installed, such as system
+  # libraries, sccache, github runners, rust, etc. This is to cut back on the amount of time it takes to launch a runner.
+  # If something is missing, it can be added here or baked into a new image outside of Github Actions.
+  plrust_arm64_amzn2:
+    name: aarch64 tests (amzn2)
+    runs-on: [self-hosted, linux, ARM64, launch_template_id__lt-0013395587950cb5d]
```

Comment:
`launch_template_id__lt-xxxxxxxxxx` is important for the new CI setup. The private ops repo has more explanation on what this means.